src/lib/util/ares_threads.c

```diff
@@ -487,6 +487,7 @@ void ares_thread_mutex_unlock(ares_thread_mutex_t *mut)
 
 struct ares_thread_cond {
   pthread_cond_t cond;
+  pthread_condattr_t attr;
```

Comment:
The pthread_condattr_t doesn't need to be saved after you've passed it to pthread_cond_init(), so please just make it a local scoped variable instead of saving it.

Reply:
Yes, i'll fix that. thanks

Reply:
ping